common/charts/couchdb/templates/statefulset.yaml

```diff
@@ -21,6 +21,18 @@ spec:
         app: {{ template "couchdb.name" . }}
         release: {{ .Release.Name }}
     spec:
+    {{- if .Values.podAntiAffinity }}
```

Comment:
Given how expressive the label selector that gets expanded is, it seems like `.Values.podAntiAffinity` doesn't really communicate the intent for this code.  I recommend finding another term. Some suggestions come to mind:

1. .Values.spreadPods
2. .Values.spreadPodsUniformly
3. .Values.distributePods

Reply:
Yeah, this seemed too confusing to me as well. I just made `affinity` fully configurable with one example of `podAntiAffinity` in default chart values.

Reply:
Nice! I think that will work quite well.